### PR TITLE
Browse view

### PR DESCRIPTION
**Pull request by mhario**

### CODE REVIEW

File: app/components/BrowseProducts.jsx [hunk 1]
```diff
@@ -1,48 +1,55 @@
...
-  } else {
-    viewProducts = props.products;
   }
 
   return (
-    <div className="products-container col-xs-12">
+    <div className="products-container">
       {
         viewProducts &&
         viewProducts.map(product => {
```
Comment (bethqiang) [inline app/components/BrowseProducts.jsx]: This should also probably be its own component?

File: app/store.jsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import thunkMiddleware from 'redux-thunk';
 
 import {whoami} from './reducers/auth';
 
-const store = createStore(rootReducer, applyMiddleware(createLogger(), thunkMiddleware));
+const store = createStore(rootReducer, applyMiddleware(createLogger({collapsed: true}), thunkMiddleware));
```
Comment (bethqiang) [inline app/store.jsx]: Are you particularly fond of the collapsed look? I'd prefer the uncollapsed look but if you're really keen on this...

File: public/stylesheets/orders.css [hunk 3]
```diff
@@ -1,36 +0,0 @@
-.order-container {
```
Comment (bethqiang) [inline public/stylesheets/orders.css]: Can you run `npm run build` again and put this stylesheet back in?

File: server/api/cart.js [hunk 4]
```diff
@@ -19,6 +19,7 @@ router.get('/', (req, res, next) => {
 router.put('/', (req, res, next) => {
   if (!req.session.cart) req.session.cart = { products: [], total: 0 };
   let found = false;
+  console.log(req.session);
```
Comment (bethqiang) [inline server/api/cart.js]: Delete this por favor!

File: server/api/products.js [hunk 5]
```diff
@@ -10,15 +11,20 @@ router.get('/', (req, res, next) => {
   .catch(next);
 });
 
-// find by id
-router.get('/:id', (req, res, next) => {
-  Products.findById(req.params.id)
-  .then(product => {
-    res.json(product);
+// load reviews by product id
+router.get('/reviews/:id', (req, res, next) => {
```
Comment (bethqiang) [inline server/api/products.js]: I almost think we should make this a query string--`/reviews?product=id` given what Omri's told us about params vs. query strings.

File: app/components/BrowseProducts.jsx (hunk 1 above)
Comment (mhario) [inline app/components/BrowseProducts.jsx]: I can do that

File: app/store.jsx (hunk 2 above)
Comment (mhario) [inline app/store.jsx]: I really don't like when each action takes like 4 lines.. I wonder if there is an easy way for us to use different versions of this file. Could .gitignore it lol

File: public/stylesheets/orders.css (hunk 3 above)
Comment (mhario) [inline public/stylesheets/orders.css]: Done, will be pushing again shortly

File: server/api/products.js (hunk 5 above)
Comment (mhario) [inline server/api/products.js]: I would rather avoid rewriting working code at the moment, I'm spending time tonight adding the reviews widget

File: app/store.jsx (hunk 2 above)
Comment (bethqiang) [inline app/store.jsx]: If we git ignore it, we won't have a store...and that would be a very big problem. It's fine, I'll live.

**Pull request merged**